### PR TITLE
[14.0][IMP] hr_expense_invoice: Copy attachments from expense to invoice

### PR DESCRIPTION
**Pull request by victoralmau**
Copy attachments from expense to invoice

Please @pedrobaeza and @chienandalu can you review it?

@Tecnativa TT41948

### CODE REVIEW

File: hr_expense_invoice/models/hr_expense.py [hunk 1]
```diff
@@ -24,6 +24,11 @@ class HrExpense(models.Model):
         copy=False,
     )
 
+    def _create_attachments_from_invoice(self, invoice):
```
Comment (pedrobaeza) [inline hr_expense_invoice/models/hr_expense.py]: No need of separate method IMO.

File: hr_expense_invoice/models/hr_expense.py [hunk 2]
```diff
@@ -24,6 +24,11 @@ class HrExpense(models.Model):
...
+        res = self.action_get_attachment_view()
```
Comment (pedrobaeza) [inline hr_expense_invoice/models/hr_expense.py]: Why do you need this action? Isn't to look for model and ID?

**Comment by nimarosa** [comment]
@victoralmau please Can you rebase this PR to 14.0 branch? So tests will rerun.

**Comment by victoralmau** [comment]
Rebase done.

File: hr_expense_invoice/models/hr_expense.py (hunk 2 above)
Comment (victoralmau) [reply hr_expense_invoice/models/hr_expense.py]: Ok, it was for re-using existing methods, although the domain is very short, it is already changed.

**Review by pedrobaeza — APPROVED** [review]

**Review by chienandalu — APPROVED** [review]

**Pull request merged**